sylvia/src/multitest.rs

```diff
@@ -1,13 +1,53 @@
 use std::cell::{Ref, RefCell, RefMut};
+use std::fmt::{Debug, Display};
 use std::marker::PhantomData;
 
-use cosmwasm_std::{Addr, BlockInfo, Coin};
-use cw_multi_test::Executor;
+use cosmwasm_std::testing::{MockApi, MockStorage};
+use cosmwasm_std::{
+    Addr, Api, BlockInfo, Coin, CustomQuery, Empty, GovMsg, IbcMsg, IbcQuery, Storage,
+};
+use cw_multi_test::{
+    BankKeeper, DistributionKeeper, Executor, FailingModule, Router, StakeKeeper, WasmKeeper,
+};
+use schemars::JsonSchema;
+use serde::de::DeserializeOwned;
 use serde::Serialize;
 
-#[derive(Default)]
-pub struct App {
-    pub app: RefCell<cw_multi_test::App>,
+/// Type alias for default build `App` to make its storing simpler in typical scenario
+pub type BasicApp<ExecC = Empty, QueryC = Empty> = App<
+    BankKeeper,
+    MockApi,
+    MockStorage,
+    FailingModule<ExecC, QueryC, Empty>,
+    WasmKeeper<ExecC, QueryC>,
+    StakeKeeper,
+    DistributionKeeper,
+    FailingModule<IbcMsg, IbcQuery, Empty>,
+    FailingModule<GovMsg, Empty, Empty>,
+>;
+
+// This exist to silence `clippy::type-complexity`.
```

Comment:
This exists to improve readability; this comment is not needed. Clippy points it out to you because it affects code clarity.